src/modules/extensible-version/ead/modules/models-and-shared-elements.rng

```diff
@@ -63,7 +63,6 @@
             <ref name="element.filePlan"/>
             <ref name="element.otherDescriptiveInfo"/>
             <ref name="element.otherFindAid"/>
-            <ref name="element.physicalOrTechnicalRequirements"/>
```

Comment:
Good catch, @deletedname !  This was indeed removed with this branch, but when I attempted to resolve a few of the merge conflicts for the all-in-one testing branch, I overwrote this removal.  I'll fix that now in the testing branch.